### DISCUSSION TITLE
<title>Github Account Suspension

### DISCUSSION THREAD

**Discussion by TheMoeedKhan** [community/community#109033] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My GitHub account got suspended suddenly.
I don't know why as I didn't receive any emails or hints indicating why this happened. I have contacted Github Support and they provided me  Ticket ID: 2602952.

![ss github suspension](https://github.com/community/community/assets/70565490/863d6f66-a79f-4c54-8970-b4da1fb576d0)

**Comment by gil-son** [comment]
You can follow the Ticket to receive some solutions
